content/docs/security/security-overview.md

```diff
@@ -40,7 +40,11 @@ Data-at-rest encryption is a method of storing inactive data that converts plain
 
 ## Secure data centers
 
-Neon’s infrastructure is hosted and managed within Amazon’s secure data centers backed by [AWS Cloud Security](https://aws.amazon.com/security/). Amazon continually manages risk and undergoes recurring assessments to ensure compliance with industry standards. For information about AWS data center compliance programs, refer to [AWS Compliance Programs](https://aws.amazon.com/compliance/programs/).
+Neon’s infrastructure is hosted and managed within either Amazon's or Azure's secure data centers, depending on the cloud service provide you selected when setting up your project.
```

Comment:
`depending on the cloud service provide` should be `provider`

Reply:
Fixed.